temporian/implementation/numpy/operators/tests/assign/assign_test.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl.testing import absltest
+
+from temporian.implementation.numpy.operators import assign
+from temporian.implementation.numpy.operators.tests.assign.test_data import (
+    different_index,
+)
+from temporian.implementation.numpy.operators.tests.assign.test_data import (
+    with_idx_more_timestamps,
+)
+
+from temporian.implementation.numpy.operators.tests.assign.test_data import (
+    with_idx_same_timestamps,
+)
+
+from temporian.implementation.numpy.operators.tests.assign.test_data import (
+    broadcast_assigned,
+)
+
+
+class AssignOperatorTest(absltest.TestCase):
+    def setUp(self) -> None:
+        self.operator = assign.NumpyAssignOperator()
+
+    def test_different_index(self) -> None:
+        self.assertRaisesRegex(
+            ValueError,
+            "Assign sequences must have the same index names.",
+            self.operator,
+            different_index.INPUT_1,
+            different_index.INPUT_2,
+        )
+
+    def test_with_idx_more_timestamps(self) -> None:
+        operator_output = self.operator(
+            with_idx_more_timestamps.INPUT_1,
+            with_idx_more_timestamps.INPUT_2,
+        )
+        self.assertEqual(
+            True,
+            with_idx_more_timestamps.OUTPUT == operator_output["event"],
+        )
+
+    def test_with_idx_same_timestamps(self) -> None:
+        operator_output = self.operator(
+            with_idx_same_timestamps.INPUT_1,
+            with_idx_same_timestamps.INPUT_2,
+        )
+        self.assertEqual(
+            True,
+            with_idx_same_timestamps.OUTPUT == operator_output["event"],
+        )
+
+    def test_broadcast_assigned(self) -> None:
```

Comment:
The word `broadcast` here might be confusing later on when we implement the broadcast operator. Maybe "test_less_index_values"?

Reply:
sure, i will change it